wheaton-college-phd-in-biblical-and-theological-studies.csl

```diff
@@ -331,8 +333,10 @@
         <text term="in" text-case="capitalize-first" suffix=" "/>
         <choose>
           <if variable="volume" match="any">
-            <text term="volume" form="short" suffix=" "/>
-            <text variable="volume" prefix=" " suffix=" of "/>
+            <group delimiter=" ">
+              <text term="volume" form="short"/>
+              <text variable="volume" suffix=" of "/>
```

Comment:
No, the commit didn't change that behaviour and it seems to conform with your examples: vol 3 of <i>Title</i>. I think @adam3smith has tought it could also be e.g. vol 3 of 6 (then the 6 = number of volumes would have been missing).